.eslintrc

```diff
@@ -40,6 +51,79 @@
           }
         ]
       }
-    }
+    },
+    {
+      // Turn off react linting rules for most packages/files 
+      "files": [
+        "spec/**",
+        "app/javascript/packs/**",
+        "app/javascript/packages/address-search/**",
+        "app/javascript/packages/components/**",
+        "app/javascript/packages/compose-components/**",
+        "app/javascript/packages/form-steps/**",
+        "app/javascript/packages/react-hooks/**",
+        "app/javascript/packages/react-i18n/**",
+        "app/javascript/packages/spinner-button/**",
+        "app/javascript/packages/step-indicator/**",
+        "app/javascript/packages/validated-field/**",
+        "app/javascript/packages/verify-flow/**",
+        // In progress: enabling these rules for all files in packages/document-capture
+        "app/javascript/packages/document-capture/context/**", 
+        "app/javascript/packages/document-capture/higher-order/**", 
+        "app/javascript/packages/document-capture/hooks/**",
+        // Comment out a file to enable react lint rules for that file only
+        "app/javascript/packages/document-capture/components/acuant-camera.tsx",
+        "app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx",
+        "app/javascript/packages/document-capture/components/acuant-capture.tsx",
+        "app/javascript/packages/document-capture/components/acuant-selfie-camera.tsx",
+        "app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.jsx",
+        //"app/javascript/packages/document-capture/components/back-button.tsx",
```

Comment:
Based on the description above, I'd expect commenting out a file would be something we only do in local development and not actually commit?

Reply:
I left these in thinking that it would give a sense of progress to see how many have been done. Looking at it again, I think the "comments in code" ick is probably not worth it. I'm going to delete them instead of commenting them.

Reply:
[Done](https://github.com/18F/identity-idp/pull/9706/commits/8625bff568f28032c2eb97113127b2f1948f280b).

Reply:
At least speaking for myself, I love to boost my 🟥 lines removed stats, so I'd have a bigger sense of accomplishment removing them than commenting them out 😄

Reply:
Good point, removing lines is infinitely more satisfying (and less messy).